### PR TITLE
Create 4.25 project plan

### PR DESCRIPTION
**Pull request by lshanmug**
Project plan for 4.25

### CODE REVIEW

File: development/plans/eclipse_project_plan_4_25.xml [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+
+<p>The release deliverables have the same form as previous releases, namely: </p>
+<ul>
+  <li>Source code release for all Eclipse Project deliverables, available as
+    versions tagged &quot;R4_23&quot; in the Eclipse Project <a href="http://git.eclipse.org">Git
+    repositories</a>.</li>
+  <li>Eclipse SDK (runtime binary and SDK for Equinox, Platform, JDT, and PDE) (downloadable).</li>
+  <li>Eclipse Equinox (runtime and source repositories) (downloadable).</li>
+  <li>Eclipse Platform (runtime and source repositories) (downloadable).</li>
+  <li>Eclipse RCP (runtime and source repositories for the Rich Client Platform) (downloadable).</li>
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_25.xml:80]: Please remove this one. It's not part of 4.24 even.

Comment (lshanmug) [reply development/plans/eclipse_project_plan_4_25.xml]: Done

**Comment by lshanmug** [comment]
The links to the subproject plans at the bottom need to be fixed.

**Comment by akurtakov** [comment]
> The links to the subproject plans at the bottom need to be fixed.

Or remove them from this document. Subprojects plans are WIP while this document is "the official" statement what we support.

**Comment by lshanmug** [comment]
> > The links to the subproject plans at the bottom need to be fixed.
> 
> Or remove them from this document. Subprojects plans are WIP while this document is "the official" statement what we support.

The subproject plans show the plan items for the release and point to the Themes and Priorities. For now, I've commented out the links and added TBD as  I was unable to find equivalent GitHub links for the plan issues. If we don't want to add the links, we have to remove the whole 'Themes and Priorities' section.

**Comment by akurtakov** [comment]
@tjwatson @jarthana @vogella What do you think?
I think removing "Themes and priorities" sections should be done as people working full time are more or less trying to enable others to contribute and fix bugs arising. Only exception is new Java version support which is being planned. 
P.S. McQ is not listed in https://github.com/orgs/eclipse-platform/teams/eclipse-platform-committers/members so he seems to haven't setup his Github account thus there is no way to mention him here.

**Comment by vogella** [comment]
Where is this document life? I checked one link in the PR  (https://www.eclipse.org/eclipse/development/porting/eclipse_4_25_porting_guide.html) and that gives me a 404

**Comment by lshanmug** [comment]
https://www.eclipse.org/projects/project-plan.php?planurl=https://www.eclipse.org/eclipse/development/plans/eclipse_project_plan_4_25.xml

**Comment by vogella** [comment]
> @tjwatson @jarthana @vogella What do you think? I think removing "Themes and priorities" sections should be done as people working full time are more or less trying to enable others to contribute and fix bugs arising. Only exception is new Java version support which is being planned. P.S. McQ is not listed in https://github.com/orgs/eclipse-platform/teams/eclipse-platform-committers/members so he seems to haven't setup his Github account thus there is no way to mention him here.

+1 for removal.

**Comment by tjwatson** [comment]
> @tjwatson @jarthana @vogella What do you think? I think removing "Themes and priorities" sections should be done as people working full time are more or less trying to enable others to contribute and fix bugs arising. Only exception is new Java version support which is being planned. P.S. McQ is not listed in https://github.com/orgs/eclipse-platform/teams/eclipse-platform-committers/members so he seems to haven't setup his Github account thus there is no way to mention him here.

+1 for removal.

**Comment by jarthana** [comment]
> @tjwatson @jarthana @vogella What do you think? I think removing "Themes and priorities" sections should be done as people working full time are more or less trying to enable others to contribute and fix bugs arising. Only exception is new Java version support which is being planned. P.S. McQ is not listed in https://github.com/orgs/eclipse-platform/teams/eclipse-platform-committers/members so he seems to haven't setup his Github account thus there is no way to mention him here.

+1 for removal

**Pull request merged**